External/SPMmodified/xASL/xASL_io_SaveNifti.m

```diff
@@ -1,4 +1,4 @@
-function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, changeMat)
+function xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew, nBits, bGZip, changeMat, bCopyOrigJson, JsonFields)
 % Save a file to a Nifti format, while taking the parameters from another file
 %
 % FORMAT: xASL_io_SaveNifti(pathOrigNifti, pathNewNifti, imNew[, nBits, bGZip, newMat])

```

Comment:
Please update the FORMAT by adding the new parameters.
Note also that newMat -> changeMat

Reply:
Fixed in 67d6fdaccd87237de371db497c72181e2c8b1c8a